go.mod

```diff
@@ -1,17 +1,19 @@
 module github.com/coreos/go-oidc/v3
 
-go 1.19
+go 1.21
```

Comment:
To the best of my knowledge, this directive describes "this package can only be used with go 1.21 and up." But this package works fine with older versions